keyboards/nopunin10did/railroad/railroad.h

```diff
@@ -0,0 +1,44 @@
+#include "quantum.h"
+
+// The Railroad Keyboard
+// Copyright 2020 by W. Alex Ronke, a.k.a. NoPunIn10Did (w.alex.ronke@gmail.com)
+// Licensed under GNU GPL v2, as per the QMK firmware project
+
+#ifndef RAILROAD_H
+#define RAILROAD_H
+
+
+#define KEYMAP( \
```

Comment:
```suggestion
#define LAYOUT( \
```

`KEYMAP` is no longer used by QMK.

Reply:
Updated manually